.github/workflows/npm.yml

```
@@ -0,0 +1,41 @@
+name: NPM
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - ".storybook/**"
+      - "src/**"
+      - "rollup.config.ts"
+      - "package.json"
+      - "yarn.lock"
+  workflow_dispatch:
+
+jobs:
+  npm-compile-publish-contracts:
```

```suggestion
  npm-compile-publish:
```
We don't deal with contracts here, we can drop that part of the job name.

---

9c7395f5e0c93e8bb91d010d3c63916bafee0e3a